sync/downstream.py

```diff
@@ -176,7 +176,13 @@ def next_action(self):
             # If we have infra failure, flag for human intervention. Retrying stability
             # runs would be very costly
             if latest_try_push.infra_fail:
-                return DownstreamAction.manual_fix
+                tasks = latest_try_push.tasks()
+
+                # Check if we had any successful tests
+                if tasks.has_completed_tests():
```

Comment:
I think this should use `tasks.wpt_tasks` to allow for the possibility of `source-test` jobs being considered tests.

Reply:
The `tasks.has_completed_tests` method filters for wpt tests like so `wpt_tasks = self.wpt_tasks.filter(tc.is_test)`. Will this filter out `source-test` jobs?

Reply:
Oh it seems like they're kind `source-test` so I guess this is fine.